src/compat/responder.js

```diff
@@ -6,13 +6,14 @@ const log = require('debug')('libp2p:mdns:compat:responder')
 const { SERVICE_TAG_LOCAL } = require('./constants')
 
 class Responder {
-  constructor (peerInfo) {
-    if (!peerInfo) {
-      throw new Error('missing peerInfo parameter')
+  constructor (peerId, multiaddrs) {
```

Comment:
Same here, make an object?
```suggestion
  constructor ({ peerId, multiaddrs }) {
```